README.md

```diff
@@ -3,29 +3,79 @@
 
 This plugin allows you to test **Electron** applications with [TestCafe](http://devexpress.github.io/testcafe).
 
-## Install
+## Getting Started
+
+### Installation
 
 ```
 npm install testcafe-browser-provider-electron
 ```
 
-## Usage
+### Testing a JavaScript Application
 
-First, create a `.testcafe-electron-rc` file in the root directory of your Electron app. For more info, see the [Configuration](#configuration) section.
```

Comment:
We can omit words like "first" without losing meaning. Short sentences are easier to read.
So I suggest to start with "Create".